src/Microsoft.ML.CodeGenerator/CodeGenerator/CSharp/CodeGenerator.cs

```diff
@@ -329,18 +329,11 @@ internal IList<(string, string[])> GenerateTransformsAndUsings(IEnumerable<Pipel
         {
             if (_pipeline == null)
                 throw new ArgumentNullException(nameof(_pipeline));
-            try
-            {
-                var node = _pipeline.Nodes.Where(t => t.NodeType == PipelineNodeType.Trainer).First();
-                ITrainerGenerator generator = TrainerGeneratorFactory.GetInstance(node);
-                var trainerString = generator.GenerateTrainer();
-                var trainerUsings = generator.GenerateUsings();
-                return (trainerString, trainerUsings);
-            }
-            catch (Exception)
-            {
-                return (string.Empty, new string[0]);
-            }
+            var node = _pipeline.Nodes.Where(t => t.NodeType == PipelineNodeType.Trainer).First();
+            ITrainerGenerator generator = TrainerGeneratorFactory.GetInstance(node);
+            var trainerString = generator.GenerateTrainer();
+            var trainerUsings = generator.GenerateUsings();
+            return (trainerString, trainerUsings);
```

Comment:
I'm not sure why the try/catch was needed. If no one else knows, we may want drill down in the code, or run a variety of datasets thru the CLI to show that nothing is expected to throw.

The old catch seem to eat the error without a hint of what went wrong.

Reply:
Yeah, that's why I removed that part of code